nearswap/src/pool.rs

```diff
@@ -60,4 +79,51 @@ impl Pool {
             total_shares: self.total_shares.into(),
         }
     }
+
+    pub fn log_observation(&mut self) {
```

Comment:
so, this should be an internal (`pub(crate)`) method of the new Twap structure.

Reply:
ok